dev

```diff
@@ -3,14 +3,28 @@
 set -e
 
 DIR=$(pwd)
-MESG_LOG_LEVEL=${MESG_LOG_LEVEL:-debug}
+
+# mesg config variables
+MESG_NAME=${MESG_NAME:-engine}
+MESG_PATH=${MESG_PATH:-$HOME/.mesg}
+MESG_SERVER_ADDRESS=${MESG_SERVER_ADDRESS:-:50052}
```

Comment:
This `MESG_SERVER_ADDRESS ` config is not useful because it's only changing the internal port of the engine server. Each engine will get a different ip address even on the same network, thus they can all expose the server on the same internal port.
Even worst, this variable is not injected in the docker service so it will never change the port of the server..